toolkit/tools/pkg/imagecustomizerlib/liveosisoimages.go

```diff
@@ -24,6 +25,15 @@ import (
 )
 
 const (
+
+	// ToDo: this is not being invoked...
```

Comment:
What does this TODO comment mean?

Reply:
I was seeing errors in the console log where the kernel fails to launch /init (the script I was creating - mimicking the toolkit).
It turns out that it would never work because our init myst be /lib/systemd/systemd - and systemd will refuse to run as a system manager unless it is pid 1. So, have a script will always fail (since bash gets pid 1).
The correct fix is to not use a script, and instead use a symbolic link from /init -> /lib/systemd/systemd.
Fixed.